README.md

```diff
@@ -195,44 +220,79 @@ All the options are supported. The callback function also receives these exports
 
 The `ConsentManager` React component is a prebuilt consent manager UI (it's the one we use on https://segment.com) that uses the [ConsentManagerBuilder][] component under the hood. To use it, just mount the component where you want the consent banner to appear and pass in your own custom copy.
 
-#### Props
+#### ConsentManager Props
+
+This section lists the configurable props you can change when
+loading the out of the box Consent Manager. In [this demo](https://codepen.io/samuelkahr/pen/eYYrobB), these are all the props you can pass into the returned preferences object.
+
+<!-- TOC depthFrom:5 depthTo:5 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [writeKey](#writekey)
+- [otherWriteKeys](#otherwritekeys)
+- [shouldRequireConsent](#shouldrequireconsent)
+- [closeBehavior](#closebehavior)
+- [implyConsentOnInteraction](#implyconsentoninteraction)
+- [defaultDestinationBehavior](#defaultdestinationbehavior)
+- [cookieDomain](#cookiedomain)
+- [bannerContent](#bannercontent)
+- [bannerSubContent](#bannersubcontent)
+- [bannerTextColor](#bannertextcolor)
+- [bannerBackgroundColor](#bannerbackgroundcolor)
+- [preferencesDialogTitle](#preferencesdialogtitle)
+- [preferencesDialogContent](#preferencesdialogcontent)
+- [cancelDialogTitle](#canceldialogtitle)
+- [cancelDialogContent](#canceldialogcontent)
+- [customCategories](#customcategories)
+-
+<!-- /TOC -->
+
 
 ##### writeKey
 
-Type: `string`
+**Type**: `string`
+**Default**: none
 
-The write key analytics.js should be loaded with.
+The Segment source-specific write key which you can find in your `analytics.load()` call in your Analytics.js snippet.
```

Comment:
I don't think it is the best way to tell the user, since as the document says at the beginning, analytics.load() has to be removed and it can lead to confusion. Maybe add a link to the segment documentation.